packages/peregrine/lib/store/actions/user/asyncActions.js

```diff
@@ -1,60 +1,22 @@
 import { Magento2 } from '../../../RestApi';
 import BrowserPersistence from '../../../util/simplePersistence';
 import { refresh } from '../../../util/router-helpers';
-import { getCartDetails, removeCart } from '../cart';
+import { removeCart } from '../cart';
 import { clearCheckoutDataFromStorage } from '../checkout';
 
 import actions from './actions';
 
 const { request } = Magento2;
 const storage = new BrowserPersistence();
 
-export const signIn = credentials =>
```

Comment:
Should have deleted this in the #1904 but I didn't. Doing it now since I'm trying to clean up refs to `getCartDetails`.